shared_model/bindings/model_crypto.cpp

```diff
@@ -21,8 +21,31 @@
 namespace shared_model {
   namespace bindings {
     crypto::Keypair ModelCrypto::generateKeypair() {
-      // TODO: 06.12.2017 luckychess create keypair generation interface IR-684
       return crypto::CryptoProviderEd25519Sha3::generateKeypair();
     }
+
+    crypto::Keypair ModelCrypto::generateKeypair(const std::string &seed) {
+      return crypto::CryptoProviderEd25519Sha3::generateKeypair(
+          crypto::Seed(seed));
```

Comment:
Maybe just remove explicit keyword in Seed's constructor? It's quite obvious from the code what's happening

Reply:
Codacy blames for one argument constructors without explicit keyword (in rare cases when it works =) )